.github/workflows/release-nightly.yml

```diff
@@ -1,8 +1,10 @@
 name: Release nightly
 
 on:
-  schedule:
-    - cron: '0 2 * * *' # run at 2 AM UTC
+  push:
+    branches:
+      - main
+      - chore/fix-nightly-builds
```

Comment:
Do we need this branch to trigger the CI?

Reply:
Resolving (duplicate comment)